packages/renderer/src/lib/recommendation/ExtensionBanners.svelte

```diff
@@ -1,10 +1,29 @@
 <script lang="ts">
+import { onDestroy, onMount } from 'svelte';
+import type { Unsubscriber } from 'svelte/store';
+
 import ExtensionBanner from '/@/lib/recommendation/ExtensionBanner.svelte';
 import { extensionBannerInfos } from '/@/stores/extensionBanners';
 
 import { isDark } from '../../stores/appearance';
+
+let isDarkUnsubscribe: Unsubscriber;
+let isDarkTheme = false;
+
+onMount(async () => {
+  isDarkUnsubscribe = isDark.subscribe(value => {
+    isDarkTheme = value;
+  });
+});
+
+// remove callback when the component is destroyed
+onDestroy(() => {
+  if (isDarkUnsubscribe) {
+    isDarkUnsubscribe();
+  }
+});
 </script>
 
 {#each $extensionBannerInfos as banner (banner.extensionId)}
-  <ExtensionBanner banner={banner} isDark={$isDark} />
+  <ExtensionBanner banner={banner} isDark={isDarkTheme} />
```

Comment:
As far as I understand svelte reactivity, it seems to me that the changes in this file are not necessary, we are just adding a layer of reactivity.
(and testing without these changes, it works the same)

Reply:
I see, thank you, I thought that I've forgot to use subscribe here